2022/31xxx/CVE-2022-31186.json

```diff
@@ -1,18 +1,101 @@
 {
```

Comment:
'https://next-auth.js.org/configuration/options#logger ' does not match '^(ftp|http)s?://\\S+$'